### PR TITLE
Adds Axios library

### PR DESCRIPTION
**Pull request by angepol**
## Context

Please link ticket here

## Pre-merge Checklist

- [x] run backend tests
- [x] run frontend tests

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -14,7 +15,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test --transformIgnorePatterns \"node_modules/(?!axios)/\"",
```
Comment (sofiapag) [inline client/package.json:18]: Is this ignoring test files in node_modules?

File: client/src/SubscriptionSection.js [hunk 2]
```diff
@@ -12,50 +13,51 @@ const SubscriptionSection = () => {
...
+        `${process.env.REACT_APP_API_BASE_URL}/create-incomplete-subscription`,
+        {
+          priceId: stripePriceId,
+        },
+        {
+          headers: { "Content-Type": "application/json" },
+        }
+      )
+      .then((response) => {
+        console.log(response);
```
Comment (sofiapag) [inline client/src/SubscriptionSection.js:49]: Let's remove that

**Review by sofiapag — APPROVED** [review]
Thanks for doing this!

File: client/package.json (hunk 1 above)
Comment (angepol) [reply client/package.json:18]: Here is context on the problem & the fix.
https://github.com/axios/axios/issues/5101

**Pull request merged**